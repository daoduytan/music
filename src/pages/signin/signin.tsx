import React, { FC } from "react";

interface Props {}

export const Signin: FC<Props> = () => {
  return <div>Signin</div>;
};

export default Signin;
