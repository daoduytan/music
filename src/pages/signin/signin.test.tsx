import { render } from "@testing-library/react";
import React from "react";
import { Signin } from "./signin";

test("Make snapshot", () => {
  const component = render(<Signin />);

  expect(component.container).toMatchSnapshot();
});

test("Render signin", () => {
  const div = document.createElement("div");

  render(<Signin />, div);
});
