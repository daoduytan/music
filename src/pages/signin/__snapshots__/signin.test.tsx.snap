// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make snapshot 1`] = `
<div>
  <div>
    Signin
  </div>
</div>
`;
